### ISSUE TITLE
Coloured book avalaibility status

### ISSUE THREAD

**Issue by Cass10p** [melmothx/mycorrhiza#10]
Make an icon (book - part of book - newspaper/review - part of newspaper/review) changing with the status of the entry (only metadata, only PDF, only TXT, PDF+TXT, translation in progress) un the search result page

**Comment by melmothx** [comment]
I think this was done in other form

**Comment by Cass10p** [comment]
Not really, because you have the filter (i think do you mean that as done in other forms), but not a fast reading icon directly on the search results
